pkg/private/pkg_files.bzl

```diff
@@ -202,6 +203,28 @@ def add_directory(content_map, dir_path, origin, mode=None, user=None, group=Non
         group = group,
     )
 
+def add_empty_file(content_map, dest_path, origin, mode=None, user=None, group=None):
+    """Add a single file to the content map.
```

Comment:
Perhaps this docstring should say "Add a single empty file to the content map"?

Reply:
Done.